specification/confidentialledger/resource-manager/readme.md

```diff
@@ -69,11 +69,14 @@ This is not used by Autorest itself.
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-java-track2
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-go-track2
   - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-js-track2
```

Comment:
don't need this, the azure-sdk-for-js is using track2 generator.

Reply:
Done